docs/modules/user-guide/partials/ref_devfile-resources.adoc

```diff
@@ -1,4 +1,4 @@
-[id="ref_devfile-samples_{context}"]
+[id="ref_devfile-resources_{context}"]
 = Devfile resources
```

Comment:
Did you create the `navtitle` attribute to replace instances of `Defile resources` like this one? Do you intend to add content between the title and the additional resources section?

Reply:
Not quite sure what you mean, Rolfe. But we simply wanted to change `samples` to `resources.` No adding content. Does that answer your question?